src/ServiceControl.Audit.Persistence.RavenDb5/DatabaseConfiguration.cs

```diff
@@ -1,25 +1,44 @@
 ﻿namespace ServiceControl.Audit.Persistence.RavenDb5
```

Comment:
The namespace doesn't match the one used below. Some types are in `ServiceControl.Audit.Persistence.RavenDb5` others in `ServiceControl.Audit.Persistence.RavenDb`

Reply:
I've removed all usages of RavenDb5 for now. We can decide at the end if we want to change all namespaces to RavenDb5 or leave as is